sdlib/core/stdc/stdio.d

```diff
@@ -1,5 +1,10 @@
 module core.stdc.stdio;
 
+// POSIX.1 (1996)
+enum stdin = 0;
+enum stdout = 1;
+enum stderr = 2;
```

Comment:
It's not even the right type: https://en.cppreference.com/w/cpp/io/c/std_streams